Maquetacion/scss/tech.scss

```diff
@@ -1,23 +1,21 @@
-.front {
-  display: inline-flex;
-  flex-wrap: wrap;
-  margin: 0 125px;
+.container {
+  display: grid;
+  grid-gap: 10px;
+  grid-template-areas: 'bigitem . .' 'bigitem . .';
+  justify-content: center;
+  margin: 30px;
+  text-align: center;
 
   .tech {
-    background-color: $green;
-    border-radius: 5px;
-    box-shadow: -10px -10px $white, -10px -10px 0 1px $green;
-    display: inline-flex;
-    flex-direction: column;
     height: 320px;
-    justify-content: center;
-    margin: 15px 15px;
-    text-align: center;
-    transition: 1s;
+    padding: 5px;
     width: 220px;
 
-    &:hover {
-      box-shadow: 10px 10px $white, 10px 10px 0 1px $green;
+    &:first-child {
+      border-right: 2px solid $green;
+      grid-area: bigitem;
+      height: 100%;
+      padding: 160px 5px;
```

Comment:
Why did you set this huge padding? If it's just for centering the content of the tech card, you can use another positioning technique based in grid/flex. :)

Reply:
the padding is corrected by a display: grid